netbox/data_source_netbox_tenants_test.go

```diff
@@ -65,3 +65,39 @@ data "netbox_tenants" "test" {
 		},
 	})
 }
+
+func TestAccNetboxTenantsDataSource_tenantgroups(t *testing.T) {
+
+	testSlug := "tnt_ds_tenant_group_filter"
+	testName := testAccGetTestName(testSlug)
+	resource.Test(t, resource.TestCase{
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+resource "netbox_tenant_group" "group_0" {
+	name = "group_%[1]s_1"
+}
+
+resource "netbox_tenant" "tenant_0" {
+  name = "tenant_%[1]s_0"
+	group_id = netbox_tenant_group.group_0.id
+}
+
+data "netbox_tenants" "test" {
+  depends_on = [netbox_tenant.tenant_0, netbox_tenant_group.group_0]
+
+	filter {
```

Comment:
this block should be once indentation (= 2 spaces) less indented

Reply:
Seems fine in VS Code?

Reply:
yes, because it is a weird inline string.
The filter attribute is on the same level as the depends_on attribute. terraform fmt` would change it, but it does not work on these kinds of strings.

Reply:
So, I'm kinda lost what to-do next to get this merged.

Reply:
remove 2 spaces from the filter block, thats all

Reply:
If I remove these 2 lines, the checks won't work any more

Reply:
nvm, I will reformat it after merging